.github/workflows/ci-model-regression.yml

```diff
@@ -0,0 +1,344 @@
+name: CI - Model Regression
+
+on:
+  push:
+    branches:
+    - '[0-9]+.[0-9]+.x'
+    tags:
+    - '**'
+  pull_request:
+    types: [opened, synchronize, labeled]
+
+env:
+  GKE_ZONE: us-central1-a
+
+jobs:
+  cleanup_runs:
+    name: Cancel old branch builds
+    runs-on: ubuntu-latest
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+
+    steps:
+      - name: Find and cancel old builds of this branch
+        uses: rokroskar/workflow-run-cleanup-action@v0.2.2
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+  read_test_configuration:
+    name: Reads tests configuration
+    if: "(github.event.pull_request.author_association == 'COLLABORATOR' || github.event.pull_request.author_association == 'MEMBER') && contains(github.event.pull_request.labels.*.name, 'status:model-regression-tests')"
```

Comment:
why do we need the author association check? should we rather check what the repo is?

Reply:
The idea behind this is to allow launch the test only for users with a specific role, but you're right, checking the repo will more elegant and the result will be the same.